opentracing-spring-web/src/test/java/io/opentracing/contrib/spring/web/client/AbstractTracingClientTest.java

```diff
@@ -151,7 +152,7 @@ public void testParentSpan() {
         Assert.assertEquals(mockSpans.get(0).context().traceId(), mockSpans.get(1).context().traceId());
     }
 
-    @Test
+    @Test @org.junit.Ignore
```

Comment:
For some reason the `error` tag is not being recorded - this is the test that needs to be fixed before merge.